yellowbrick/text/dispersion.py

```diff
@@ -258,9 +314,8 @@ def finalize(self, **kwargs):
         -----
         Generally this method is called from show and not directly by the user.
         """
-
+        self.set_title("Lexical Dispersion Plot")
         self.ax.set_ylim(-1, len(self.indexed_words_))
-        self.ax.set_title("Lexical Dispersion Plot")
```

Comment:
So much enhancement for the DispersionPlot for just a fix of this line! Thank you so much for all the effort you put into it!